org.clementine_player.Clementine.yaml

```diff
@@ -0,0 +1,262 @@
+app-id: org.clementine_player.Clementine
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: clementine
+rename-desktop-file: clementine.desktop
+rename-icon: clementine
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+  - --filesystem=xdg-music
+  - --persist=.config/Clementine
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=org.mpris.MediaPlayer2.clementine
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/doc
+  - /share/gtk-doc
+  - /share/info
+  - /share/man
+
+modules:
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.0.0.json
+  - shared-modules/udev/udev-175.json
+
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app
+      - ./b2 headers
+      - cp -r boost /app/include/boost
+    sources:
+      - type: archive
+        url: https://dl.bintray.com/boostorg/release/1.64.0/source/boost_1_64_0.tar.bz2
+        sha256: 7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332
+
+  - name: protobuf
+    config-opts:
+      - --with-zlib
+      - --with-pic
+    sources:
+      - type: archive
+        url: https://github.com/protocolbuffers/protobuf/releases/download/v3.6.1/protobuf-all-3.6.1.tar.gz
+        sha256: fd65488e618032ac924879a3a94fa68550b3b5bcb445b93b7ddf3c925b1a351f
+    cleanup:
+      - /bin
+      - /lib/*.a
+      - /lib/*.la
+
+  - name: fftw3
+    buildsystem: autotools
+    config-opts:
+      - --enable-shared
+      - --enable-float
+      - --enable-threads
+      - --enable-openmp
+      - --disable-fortran
+    sources:
+      - type: archive
+        url: http://fftw.org/fftw-3.3.8.tar.gz
+        md5: 8aac833c943d8e90d51b697b27d4384d
+    cleanup:
+      - /bin
+      - /lib/*.a
+      - /lib/*.la
```

Comment:
Since so many modules clean up `*.a` and `*.la` this makes sense to move to the toplevel instead.